common/djangoapps/pipeline_mako/templates/mako/inline_js.html

```diff
@@ -1,9 +1,10 @@
+<%page expression_filter="h"/>
 <script \
-% if async:
+% if context.get('async', default=None):
 async \
 % endif
 if defer:
 defer \
 type="text/javascript" charset="utf-8">
-    ${source | safe}
+    ${source}
```

Comment:
I think you need to instead switch this to `${HTML(source)}` as mentioned as one option in the docs for `mako-invalid-html-filter`.  This is only used for loading JS files from the filesystem and putting their content inline in the rendered page, so I'm pretty sure this needs to stay unescaped.  (@robrap , does any of that sound incorrect to you?)